.github/workflows/manual.yml

```
@@ -0,0 +1,71 @@
+name: "Manual Deploying Rollup Data Availability"
+
+on: workflow_dispatch
+
+jobs:
+  git-submodules-update:
+    name: "Build and deploy Rollup Data Availability"
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write        
```

yamllint

[error] trailing spaces

(trailing-spaces)